boards/x86/qemu_x86/qemu_x86.dts

```diff
@@ -4,7 +4,7 @@
 
 #include <mem.h>
 
-#define DT_FLASH_SIZE		DT_SIZE_K(1024)
+#define DT_FLASH_SIZE			DT_SIZE_K(4092)
```

Comment:
Not 4096?  Are those extra 4k doing something?  If this isn't a typo and this is intending to reserve a page for some purpose, probably wants a comment right here to warn off people like me who'd assume it's a typo.